PatientWebApplication/SearchMicroserviceApi/Model/Medicine.cs

```diff
@@ -0,0 +1,59 @@
+/***********************************************************************
+ * Module:  Medicine.cs
+ * Author:  Lidija
+ * Purpose: Definition of the Class Manager.Medicine
+ ***********************************************************************/
+
+
+using System.Collections.Generic;
+
+
+namespace SearchMicroserviceApi.Model
+{
+    public class Medicine : Equipment
+    {
+        public List<ModelRoom> room { get; set; }
+        public int doctorId { get; set; }
+        public string description { get; set; }
+        public bool isConfirmed { get; set; }
```

Comment:
Write class properties to start with upper letter and remove "this" in constructors